client/my-sites/customer-home/main.jsx

```diff
@@ -157,51 +163,98 @@ class Home extends Component {
 							{ translate( 'Review and update my site' ) }
 						</h6>
 						<div className="customer-home__card-button-pair">
-							<Button href={ site.URL } primary>
+							<Button
+								href={ site.URL }
+								primary
+								onClick={ () => trackAction( 'my_site', 'view_site' ) }
+							>
 								{ translate( 'View Site' ) }
 							</Button>
-							<Button href={ customizeUrl }>{ translate( 'Edit Homepage' ) }</Button>
+							{ isStaticHomePage ? (
+								<Button
+									href={ customizeUrl }
+									onClick={ () => trackAction( 'my_site', 'edit_homepage' ) }
+								>
+									{ translate( 'Edit Homepage' ) }
+								</Button>
+							) : (
+								<Button
+									onClick={ () => {
+										trackAction( 'my_site', 'write_post' );
+										page( `/post/${ siteSlug }` );
+									} }
+								>
+									{ translate( 'Write Blog Post' ) }
+								</Button>
+							) }
 						</div>
 					</Card>
 					<Card className="customer-home__card-boxes">
 						<div className="customer-home__boxes">
 							<ActionBox
-								onClick={ () => page( `/page/${ siteSlug }` ) }
+								onClick={ () => {
+									trackAction( 'my_site', 'add_page' );
+									page( `/page/${ siteSlug }` );
+								} }
 								label={ translate( 'Add a page' ) }
 								iconSrc="/calypso/images/customer-home/page.svg"
 							/>
-							<ActionBox
-								onClick={ () => page( `/post/${ siteSlug }` ) }
-								label={ translate( 'Write blog post' ) }
-								iconSrc="/calypso/images/customer-home/post.svg"
-							/>
+							{ isStaticHomePage ? (
+								<ActionBox
+									onClick={ () => {
+										trackAction( 'my_site', 'write_post' );
+										page( `/post/${ siteSlug }` );
+									} }
+									label={ translate( 'Write blog post' ) }
+									iconSrc="/calypso/images/customer-home/post.svg"
+								/>
+							) : (
+								<ActionBox
+									onClick={ () => {
+										trackAction( 'my_site', 'manage_comments' );
+										page( `/comments/${ siteSlug }` );
+									} }
+									label={ translate( 'Manage comments' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **140** times:
`translate( 'Manage Comments' )` *ES Score: 9*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).